docs/update.md

```diff
@@ -1,58 +1,125 @@
 
 The list of recommended upgrade scenarios includes two variants:
 
 * Upgrade to the new versions of the Operator *and* Percona XtraDB Cluster,
 * Minor Percona XtraDB Cluster version upgrade *without* the Operator upgrade.
 
-## Upgrading the Operator and CRD
+## Upgrade the Operator and CRD
 
-!!! note
+### Considerations
 
-    The Operator supports **last 3 versions of the CRD**, so it is technically
-    possible to skip upgrading the CRD and just upgrade the Operator. If the CRD
-    is older than the new Operator version *by no more than three releases*, you
-    will be able to continue using the old CRD and even carry on Percona XtraDB
-    Cluster minor version upgrades with it. But the recommended way is to update
-    the Operator *and* CRD.
+1. The Operator version has three digits separated by a dot (`.`) in the format `major.minor.patch`. Here's how you can understand the version `1.16.1`:
 
-Only the incremental update to a nearest version of the
-Operator is supported (for example, update from 1.4.0 to 1.5.0). To update
-to a newer version, which differs from the current version by more
-than one, make several incremental updates sequentially.
+    * `1` is the major version 
+    * `16` is the minor version
+    * `1` is the patch version.     
 
-!!! warning
+    You can only upgrade the Operator to the nearest `major.minor` version (for example, from `1.15.1` to `1.16.1`).     
+
+    If the your current Operator version and the version you want to upgrade to differ by more than one minor version, you need to upgrade step by step. For example, if your current version is `1.14.x` and you want to move to `1.16.x`, first upgrade to `1.15.x`, then to `1.16.x`.    
+
+    Patch versions don't influence the upgrade, so you can safely move from `1.15.1` to `1.16.1`. 
+
+    Check the [Release notes index](ReleaseNotes/index.md) for the list of the Operator versions.
+
+2. CRD supports the **last 3 minor versions of the Operator**. This means it is
+compatible with the newest Operator version and the two older minor versions.
+If the Operator is older than the CRD *by no more than two versions*, you
+should be able to continue using the old Operator version.
+But updating the CRD *and* Operator is the **recommended path**. 
+
+3. Starting with version 1.12.0, the Operator no longer has a separate API version for each release in CRD. Instead, the CRD has the API version `v1`. Therefore, if you installed the CRD when the Operator version was **older than 1.12.0**, you must update the API version in the CRD manually to run the upgrade. To check your CRD version, use the following command:
+
+    ```{.bash data-prompt="$"}
+    $ kubectl get crd perconaxtradbclusters.pxc.percona.com -o yaml | yq .status.storedVersions
+    ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        - v1-11-0
+        - v1
+        ```
```

Comment:
Maybe we also need to say "if you have anything other than `v1`, you need to perform this operation